pymc_marketing/mmm/data_generator.py

```diff
@@ -0,0 +1,611 @@
+import warnings
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+from pymc_marketing.mmm.transformers import (
+    delayed_adstock,
+    geometric_adstock,
+    logistic_saturation,
+)
+
+warnings.filterwarnings("ignore", category=FutureWarning)
```

Comment:
We shouldn't have a blank warning filter like this. You can suppress specific warnings around specific function calls instead with a local `with warnings.catch_warnings()`